stylesheets/scss/includes/_typography.scss

```diff
@@ -0,0 +1,34 @@
+@mixin headingtype($type: heading1) {
```

Comment:
@nmwd  Do you think it makes sense to add a few comments here about how one should go about adding things to this?  I'm thinking a bare minimum would be something about how the 'small' 'smaller' keys need to match up to the breakpoints map-get stuff.